webui/react/src/components/HyperparameterSearchModal.module.scss

```
@@ -69,6 +66,9 @@
         grid-auto-rows: max-content;
         overflow-x: auto;
 
+        div[class*='ant-form-item-explain'] {
```

can we just use a class selector? this is only 1 point more specific than `.ant-form-item-explain` but will make our lives harder when trying to search for specific css selectors in the future.

---

ah i think i got it, try 
```suggestion
        :global(.ant-form-item-explain) {
```
otherwise the class will be hashed.